.github/workflows/tests.yml

```diff
@@ -201,7 +201,7 @@ jobs:
 
       - name: Plugin System Tests
         if: matrix.build_type == 'system' && matrix.target == 'plugins'
-        run: LOAD_PLUGINS=1 PARALLEL_TEST_PROCESSORS=4 bin/turbo_rspec --verbose plugins/*/spec/system
+        run: LOAD_PLUGINS=1 PARALLEL_TEST_PROCESSORS=12 bin/turbo_rspec --verbose plugins/*/spec/system
```

Comment:
I think this will always have to be half of the available vCPUs since each RSpec process will need to launch an additional chrome process as well.